src/ralph/ssl_certificates/management/commands/import_ssl_certificates.py

```diff
@@ -39,36 +104,11 @@ def handle(self, *args, **options):
                         pem_data.encode(), default_backend()
                     )
                 except ValueError:
-                    sys.stderr.write(
+                    self.stderr.write(
                         '{}/{} is not valid\n'.format(root, filename)
                     )
                     continue
-
                 if cert.not_valid_after < datetime.datetime.now():
                     continue
-                try:
-                    extension = cert.extensions.get_extension_for_oid(
-                        ExtensionOID.SUBJECT_ALTERNATIVE_NAME
-                    )
-                except ExtensionNotFound:
-                    pass
-
-                if extension and extension.value:
-                    san = extension.value.get_values_for_type(x509.DNSName)
-                issuer = cert.issuer.get_attributes_for_oid(
-                    NameOID.ORGANIZATION_NAME
-                )
-                domain = Path(filename).stem
-                issuer_name = 'CA ENT'
-                if issuer and issuer[0].value:
-                    issuer_name = issuer[0].value
-                manufacturer, _ = Manufacturer.objects.get_or_create(
-                    name=issuer_name
-                )
-                SSLCertificate.objects.get_or_create(
-                    name=domain,
-                    date_to=cert.not_valid_after,
-                    date_from=cert.not_valid_before,
-                    san=san,
-                    issued_by=manufacturer
-                )
+                san = self.get_san(cert)
```

Comment:
can't `ssl_values` call this method inside?

Reply:
I'll add it into ssl_values